src/platform/forms-system/src/js/review/submit-states/ClientError.jsx

```diff
@@ -13,7 +13,7 @@ export default function ClientError(props) {
   const { buttonText, formConfig, onBack, onSubmit, testId } = props;
   const Element = Scroll.Element;
   const scrollToError = () => {
-    scrollTo('errorScrollElement', window.VetsGov?.scroll || '');
+    scrollTo('errorScrollElement', window.VetsGov?.scroll || undefined);
```

Comment:
I don't think the explicit `undefined` is necessary. Due to the optional chaining in `window.VetsGov?.scroll`, if there isn't a `scroll` property then that should resolve to `undefined`.

[Optional chaining docs](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining)

> instead of causing an error if a reference is nullish (`null` or `undefined`), the expression short-circuits with a return value of `undefined`

Reply:
I'm going to hold making any adjustments to these. There was another request for additional refinement having to do with these and once that's merged, this will be handled by the `getScrollOptions` function. Any inline declarations like this will be replaced with that function so this will be a non-issue.